### ISSUE TITLE
fusedlayernorm does not work with elementwise_affine=False

### ISSUE THREAD

**Issue by seongwook-ham** [NVIDIA/apex#403]
in distributed training setting with apex disitributed data parallel, when fused layernorm is used with elementwise_affine=False, doing convolution or elementwise sum or elementwise multiplication after fused layer norm throw following error
`RuntimeError: CUDA error: invalid configuration argument`
but torch nn layernorm works properly with elementwise_affine=False in same setting
also if i use elementwise_affine=True with fused layer norm in same setting, it works properly.
what is problem?

**Comment by ngimel** [comment]
Does it reproduce only with distributed, or it can be reproed with single GPU runs also?

**Comment by seongwook-ham** [comment]
yes single gpu case without distributed_data parallel or nn data parallel also throw same error

**Comment by ngimel** [comment]
Reopening, as it still might not be fixed.
